{"uuid": "547b6a53-db9e-4219-a9da-60fed44af605", "children": ["c220d164-f040-4175-903e-8f5cc3d2d5db", "16f161c4-a6d0-4282-825e-c206a0a02dc1", "c986263d-0856-42ad-9cf6-487b0a862e6f", "58743e78-16e6-4e6e-9d9d-6771ce99f247", "e19a1b48-719f-415b-a22b-96e61218ed8d", "f3e35447-e1f2-48f2-a4e3-dd95d6ad3c8d", "5048616c-d022-4f47-9abc-b8f25ba77835", "70e3616c-89aa-4f90-aa8c-0579c7ab5633"], "befores": [{"name": "reset_mocks", "status": "passed", "start": 1648636890840, "stop": 1648636890840}], "afters": [{"name": "reset_mocks::0", "status": "broken", "statusDetails": {"message": "ConnectionRefusedError: [Errno 61] Connection refused\n", "trace": "  File \"/Users/elenakulgavaya/work/idiomatic/qa/.venv/lib/python3.9/site-packages/allure_commons/_allure.py\", line 200, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/Users/elenakulgavaya/work/idiomatic/qa/.venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 941, in _teardown_yield_fixture\n    next(it)\n  File \"/Users/elenakulgavaya/work/idiomatic/qa/tests/tagging_ui/conftest.py\", line 18, in reset_mocks\n    MockServer().reset(name='tagging')\n  File \"/Users/elenakulgavaya/work/idiomatic/qa/qa/tools/mock.py\", line 25, in reset\n    self.get_client(name).reset()\n  File \"/Users/elenakulgavaya/work/idiomatic/qa/qa/tools/mock.py\", line 17, in get_client\n    return HTTPMock(\n  File \"/Users/elenakulgavaya/work/idiomatic/qa/.venv/lib/python3.9/site-packages/pretenders/client/http.py\", line 49, in __init__\n    super(HTTPMock, self).__init__(host, port, timeout, name)\n  File \"/Users/elenakulgavaya/work/idiomatic/qa/.venv/lib/python3.9/site-packages/pretenders/client/__init__.py\", line 102, in __init__\n    self.pretender_details = self._request_mock_access()\n  File \"/Users/elenakulgavaya/work/idiomatic/qa/.venv/lib/python3.9/site-packages/pretenders/client/__init__.py\", line 152, in _request_mock_access\n    response, data = self.boss_access.http('POST',\n  File \"/Users/elenakulgavaya/work/idiomatic/qa/.venv/lib/python3.9/site-packages/pretenders/client/__init__.py\", line 39, in http\n    response = self._get_response(method, *args, **kwargs)\n  File \"/Users/elenakulgavaya/work/idiomatic/qa/.venv/lib/python3.9/site-packages/pretenders/client/__init__.py\", line 27, in _get_response\n    self.connection.request(method=method, *args, **kwargs)\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/http/client.py\", line 1285, in request\n    self._send_request(method, url, body, headers, encode_chunked)\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/http/client.py\", line 1331, in _send_request\n    self.endheaders(body, encode_chunked=encode_chunked)\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/http/client.py\", line 1280, in endheaders\n    self._send_output(message_body, encode_chunked=encode_chunked)\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/http/client.py\", line 1040, in _send_output\n    self.send(msg)\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/http/client.py\", line 980, in send\n    self.connect()\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/http/client.py\", line 946, in connect\n    self.sock = self._create_connection(\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/socket.py\", line 844, in create_connection\n    raise err\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/socket.py\", line 832, in create_connection\n    sock.connect(sa)\n"}, "start": 1648636891246, "stop": 1648636891247}], "start": 1648636890840, "stop": 1648636891247}